autoload/gtfo/open.vim

```diff
@@ -179,7 +179,7 @@ func! gtfo#open#term(dir, cmd) abort "{{{
     if !s:empty(s:termpath)
       silent call system(s:termpath." ".shellescape(l:dir))
     elseif executable('gnome-terminal')
-      silent call system('gnome-terminal --window -e "$SHELL -c \"cd '''.l:dir.''' ; exec $SHELL\""')
+      silent call system('gnome-terminal --app-id=org.gnome.Terminal --window --working-directory "'. l:dir . '"')
```

Comment:
would prefer to keep `'''.l:dir.'''` to avoid edge cases with `$` or other weird directory names.

Reply:
OK.